tests/unit/Core/Usecase/Post/ExportTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: rowasc
+ * Date: 5/2/18
+ * Time: 1:36 PM
+ */
+
+namespace Tests\Unit\Core\Usecase\Post;
+
+use Tests\TestCase;
+use Mockery as M;
+use Ushahidi\App\Repository\ExportJobRepository;
+use Ushahidi\App\Repository\Form\AttributeRepository;
+use Ushahidi\App\Repository\Post\ExportRepository;
+use Tests\Unit\Core\Entity\MockPostEntity;
+use Tests\Unit\Core\Entity\MockExportJobEntity;
+use Ushahidi\Core\SearchData;
+
+class ExportTest extends TestCase
+{
+	public function setUp()
+	{
+		parent::setup();
+
+		$this->postExportRepository = M::mock(ExportRepository::class);
+		$this->exportJobRepository = M::mock(ExportJobRepository::class);
+		$this->formAttributeRepository = M::mock(AttributeRepository::class);
+		$this->usecase = service('factory.usecase')->get('posts_export', 'export');
+
+		// Get the usecase and pass in authorizer, payload and transformer
+		$this->usecase = $this->usecase
+			->setAuthorizer(service('authorizer.export_job'))
+			->setFormatter(service('formatter.entity.post.csv'));
+		$this->usecase->setExportJobRepository($this->exportJobRepository);
+		$this->usecase->setFormAttributeRepository($this->formAttributeRepository);
+		$this->usecase->setPostExportRepository($this->postExportRepository);
+	}
+
+	public function tearDown()
+	{
+		parent::tearDown();
+	}
+
+	public function testJobIsUpdated()
+    {
+		// set CLI params to be the payload for the usecase
+		$payload = [
+			'job_id' => 1,
+			'limit' => 100,
+			'offset' => 0,
+			'add_header' => true,
+		];
+
+		$post1 = new MockPostEntity();
```

Comment:
In most cases its much easier and just as effective to just use the Entity as-is rather than mocking it. ie. if you just use `new Post()` here instead, does that just work?